### ISSUE TITLE
Add arbitrary words to TFIDF matrix

### ISSUE THREAD

**Issue by juanshishido** [juanshishido/text-classification#1]
The test set will contain words not found in the training set (and vice versa). To calculate the cosine similarity, however, _all_ **words** (minus English stop words) need to be in both the test and training tfidf matrices.
